README.md

```diff
@@ -631,7 +631,9 @@ By default youtube-dl tries to download the best available quality, i.e. if you
 
 But sometimes you may want to download in a different format, for example when you are on a slow or intermittent connection. The key mechanism for achieving this is so-called *format selection* based on which you can explicitly specify desired format, select formats based on some criterion or criteria, setup precedence and much more.
 
-The general syntax for format selection is `--format FORMAT` or shorter `-f FORMAT` where `FORMAT` is a *selector expression*, i.e. an expression that describes format or formats you would like to download.
+The general syntax for format selection is `--format FORMAT` or shorter `-f FORMAT` where `FORMAT` is a *selector expression*, i.e. an expression that describes format or formats you would like to download.  
+
+To avoid the case where the audio and video are incompatible, specify both the format of the audio and video streams explicitly.  For example specifying `-f mp4` will default to `-f mp4,best`, but the best audio youtube can provide cannot be contained in an `MP4` file, so youtube-dl will try to merge them into an `mkv` instead of the requested `mp4` format.  To avoid this, specify `-f mp4,mp4`, so the audio and video are compatible.
```

Comment:
Completely wrong. `-f mp4` **will not** default to `-f mp4,best`.

>best audio youtube can provide cannot be contained in an `MP4` file

This is wrong as well. Some videos have `mp4` `bestaudio` some `webm`. Same goes for `bestvideo`.

>youtube-dl will try to merge them into an `mkv` instead of the requested `mp4` format

Wrong again. It will merge into `mkv` only if `bestaudio` and `bestvideo` formats are requested and they happen to be incompatible (`mp4+webm`). This is because it's impossible to mux `webm` into `mp4` container. With `-f mp4` you request single file audio-video media that **does not involve** muxing at all. Have you even read format selection section in FAQ?

>To avoid this, specify `-f mp4,mp4`, so the audio and video are compatible.

This is nonsense. `,` operator allows to specify **several formats** to be downloaded **separately** so `mp4,mp4` does not make any sense at all since it will try download best single file audio-video mp4 format and then will try to download it again.